src/main/java/com/EstimatedDeliveryDate/Entity/Holiday.java

```diff
@@ -0,0 +1,36 @@
+package com.EstimatedDeliveryDate.Entity;
+
+import java.util.List;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+
+/*EDD_HOLIDAYS table item is mapped to Holiday object*/
+
+@DynamoDBTable(tableName = "EDD_HOLIDAYS")
+public class Holiday {
+	private String SHIPPING_SERVICE_ID;
+	private List<HolidayList> HOLIDAY_DATES_LIST;
```

Comment:
Holiday list being a part of holiday class looks odd to me. Shouldn't it be the other way around?
It can be something like Holidays and holiday classes.